src/App/Command/Release/MakeCommand.php

```diff
@@ -306,13 +306,20 @@ private function displayReleaseSummary(Package $package, ComposerConfig $compose
 
         $changes = implode("\n", $changes);
 
+        $upgradeNotesLink = '';
+        $upgradePath = $package->getPath() . '/UPGRADE.md';
+        if (file_exists($upgradePath)) {
+            $upgradeNotesLink = "\n\nSee [UPGRADE.md](https://github.com/yiisoft/$packageName/blob/$versionToRelease/UPGRADE.md) for upgrade notes.";
```

Comment:
It would be good if we link to exact notes with anchor. For example:
```suggestion
            $upgradeNotesLink = "\n\nSee [UPGRADE.md](https://github.com/yiisoft/$packageName/blob/$versionToRelease/UPGRADE.md#1-0) for upgrade notes.";
```